cypress/support/pages/home.pageObject.js

```diff
@@ -3,8 +3,23 @@ import PageObject from '../PageObject';
 class homePageObject extends PageObject {
   url = '/#/';
 
-  get usernameLink() {
-    return cy.getByDataCy('/profile/');
+  usernameLink() {
+    cy.getByDataCy('/profile/')
```

Comment:
it doesn't look like data-cy attribute. please use it with data-cy attr only. as the name of the method says.
check and fix where it is needed